config.js

```diff
@@ -0,0 +1 @@
+module.exports.USDA_TOKEN = 'fLnoQvpg17ECECAj8NiUj37R9O2rvBEnYEWCCRH4';
```

Comment:
need to take out

Reply:
delete this file and add it to the gitignore
and hope a bot didn't already scrape this key

Reply:
added!